tutorials/basics/4_batch.py

```diff
@@ -1,39 +1,39 @@
 """
 .. currentmodule:: dgl
 
-Batched Graph Classification with DGL
+Tutorial: Batched graph classification with DGL
 =====================================
 
 **Author**: `Mufei Li <https://github.com/mufeili>`_,
 `Minjie Wang <https://jermainewang.github.io/>`_,
 `Zheng Zhang <https://shanghai.nyu.edu/academics/faculty/directory/zheng-zhang>`_.
 
+In this tutorial, you learn how to use DGL to batch multiple graphs of variable size and shape. The 
+tutorial also demonstrates training a graph neural network for a simple graph classification task.
+
 Graph classification is an important problem
-with applications across many fields -- bioinformatics, chemoinformatics, social
-network analysis, urban computing and cyber-security. Applying graph neural
-networks to this problem has been a popular approach recently (
+with applications across many fields, such as bioinformatics, chemoinformatics, social
+network analysis, urban computing, and cyber-security. Applying graph neural
+networks to this problem has been a popular approach recently. This can be seen in the following reserach references: 
 `Ying et al., 2018 <https://arxiv.org/abs/1806.08804>`_,
 `Cangea et al., 2018 <https://arxiv.org/abs/1811.01287>`_,
 `Knyazev et al., 2018 <https://arxiv.org/abs/1811.09595>`_,
 `Bianchi et al., 2019 <https://arxiv.org/abs/1901.01343>`_,
 `Liao et al., 2019 <https://arxiv.org/abs/1901.01484>`_,
 `Gao et al., 2019 <https://openreview.net/forum?id=HJePRoAct7>`_).
 
-This tutorial demonstrates:
- * batching multiple graphs of variable size and shape with DGL
- * training a graph neural network for a simple graph classification task
 """
 
 ###############################################################################
-# Simple Graph Classification Task
+# Simple graph classification task
 # --------------------------------
-# In this tutorial, we will learn how to perform batched graph classification
-# with dgl via a toy example of classifying 8 types of regular graphs as below:
+# In this tutorial, you learn how to perform batched graph classification
+# with DGL. The example task objective is to classify eight types of regular graphs shown here.
```

Comment:
Actually, regular graph refers to a graph where all nodes have the same degree ([link](https://en.wikipedia.org/wiki/Regular_graph)). How about changing it to topology?

Reply:
@jermainewang Thanks for your comment. Can you clarify where to modify the newly proposed text?  This? The example task objective is to classify eight types of topologies shown here.

Reply:
Exactly yes!